lib/ratchet/transformer.rb

```diff
@@ -16,20 +16,32 @@ def on_nut_attrs(attributes)
 
     private
 
-    def build_content(scope, property, children)
-      if property
-        [
-          :multi,
-          [:code, "#{property} = #{scope}[#{property.inspect}]"],
-          [:code, "if #{property}.is_a?(Hash) or #{property}.nil?"],
-          compile(children),
-          [:code, 'else'],
-          [:dynamic, property],
-          [:code, 'end'],
-        ]
-      else
-        compile(children)
-      end
+    def build_dynamic(scope, property, tag, attributes, children)
+      [
+        :multi,
+        [:code, "[#{scope}[#{property.inspect}]].flatten.each do |#{property}|"],
```

Comment:
Unfortunately `Array(hash)` results in something like `[[key, value]]`. Bummer, because I would have loved it to be that easy... Not sure how foolproof this implementation is, but I'll keep iterating... 😏